Detectors/TRD/reconstruction/include/TRDReconstruction/CruRawReader.h

```diff
@@ -76,6 +76,7 @@ class CruRawReader
     mEnableTimeInfo = options[TRDEnableTimeInfoBit];
     mEnableStats = options[TRDEnableStatsBit];
     mOptions = options;
+    mTimeBins = constants::TIMEBINS; // set to value from constants incase the DigitHCHeader1 header is not present.
```

Comment:
The header should always be present. It should be a (fatal) error if it is missing. But if this is the 'old' raw reader, it might make sense to keep this option, for the new one, we should probably remove it.

Reply:
Well I dont want the thing to crash. Bad data should get flagged and acted on. The flagging is done somewhere else. DigitHCHeader1 is optional though, only the DigitHCHeader is not optional according to the code, as I understand it. Yes I agree its pretty useless with out.